lib/ldp.js

```diff
@@ -10,6 +10,7 @@ const utils = require('./utils')
 const error = require('./http-error')
 const stringToStream = require('./utils').stringToStream
 const serialize = require('./utils').serialize
+const overQuota = require('./utils').overQuota
```

Comment:
Note for vNext: we should probably be able to implement this with a decorator pattern only.

Reply:
Sounds good. For example with traits?